runtime/lib/ttnn/operations/conv/conv2d.cpp

```diff
@@ -19,6 +20,7 @@ void run(const ::tt::target::ttnn::Conv2dOp *op, ProgramContext &context) {
   auto config = ::ttnn::operations::conv::conv2d::Conv2dConfig();
   config.dtype = utils::getDataType(op->input());
   config.weights_dtype = utils::getDataType(op->weight());
+  config.shard_layout = TensorMemoryLayout(op->shard_layout());
```

Comment:
I don't think this is a safe thing to do.  If metal ever changed their enum values we'd be broken.